cypress/e2e/shared/notificationEndpoints.test.ts

```diff
@@ -178,6 +178,10 @@ describe('Notification Endpoints', () => {
         'A minute, an hour, a month. Notification Endpoint is certain. The time is not.'
       const newURL = 'many-faced-god.gov'
 
+      cy.intercept('PUT', `/api/v2/notificationEndpoints/${endpoint.id}`).as(
+        'putEndp'
```

Comment:
Please name this something which makes sense when looking at `cy.wait` and have no context of where this is defined.